mkdocs.yml

```diff
@@ -286,12 +304,45 @@ plugins:
   - redirects:
       redirect_maps:
          jobs.md: https://jobs.applied-llms.org/
+         hub/single_classification.md: integrations/single_classification.md
```

Comment:
hub/single_classification.md: integrations/single_classification.md
         hub/multiple_classification.md: integrations/multiple_classification.md
         hub/tables_from_vision.md: integrations/tables_from_vision.md
         hub/pandas_df.md: integrations/pandas_df.md
         hub/batch_classification_langsmith.md: integrations/batch_classification_langsmith.md
         hub/action_items.md: integrations/action_items.md
         hub/partial_streaming.md: integrations/partial_streaming.md
         hub/extract_contact_info.md: integrations/extract_contact_info.md
         hub/knowledge_graph.md: integrations/knowledge_graph.md
         hub/youtube_clips.md: integrations/youtube_clips.md
         
         
         
         should be in hub